src/vma/sock/sock-redirect.cpp

```diff
@@ -541,15 +541,18 @@ extern "C"
 int vma_dump_fd_stats(int fd, int log_level)
 {
 #ifdef DEFINED_VMAPOLL
+NOT_IN_USE(fd);
+NOT_IN_USE(log_level);
 	return 0;
-#endif // DEFINED_VMAPOLL
+#else
```

Comment:
@igor-ivanov Is this fix related to page faults? If not - would you consider a separate commit?